### PR TITLE
Feat: Added hide pass, name field, header name

### PR DESCRIPTION
**Pull request by zoef-yt**
In this pr, I added

- [x] Added show password
- [x] Added test login
- [x] Added username letter on the header

### CODE REVIEW

File: src/components/css/modal.css [hunk 1]
```diff
@@ -15,7 +15,7 @@
 }
 
 .input-group .text-field {
-	margin: 0 0 1rem 0;
+	margin: 0;
 	width: 30rem;
 	max-width: 100%;
```
Comment (muskanbagrecha) [inline src/components/css/modal.css:20]: Why is this max-width:100% and width at 30rem?

File: src/components/header/header.jsx [hunk 2]
```diff
@@ -41,10 +41,7 @@ const Header = () => {
 				<div onClick={toggleTheme}>{theme === 'dark' ? <SunIcon className='header-icon' /> : <HalfMoonIcon className='header-icon' />}</div>
 				{user != null ? (
 					<div className='badge-holder'>
-						<img
-							className='avatar avatar-sm'
-							src='https://media-exp1.licdn.com/dms/image/C4E03AQEm0ZBjaIr1hg/profile-displayphoto-shrink_200_200/0/1610261504692?e=1649894400&v=beta&t=kKO77EwqflEDsrZf5eG7xC7ZUB4hD_BkZuRJyFiFHbI'
-						/>
+						<div className='avatar avatar-sm '>{user.name.substring(0, 1).toUpperCase()}</div>
```
Comment (muskanbagrecha) [inline src/components/header/header.jsx:44]: You can check if just giving the 0th index will be better or using the substring method.

**Review by iRohitGaur — APPROVED** [review]
clean and understandable code. Good to merge 👍🏻

File: src/components/modal/LoginCardModal.jsx [hunk 3]
```diff
@@ -85,7 +103,7 @@ const LoginCardModal = () => {
 					/>
 					<InputField
 						labelText='Password'
-						type='password'
+						type={!showPassword ? 'password' : 'text'}
```
Comment (muskanbagrecha) [inline src/components/modal/LoginCardModal.jsx]: Please maintain one more field in the state object to hide and show the confirm password field. 
Please enable hide and show password on login.

**Review by muskanbagrecha — CHANGES_REQUESTED** [review]
Looks good. Please incorporate the requested changes.

File: src/components/header/header.jsx (hunk 2 above)
Comment (iRohitGaur) [reply src/components/header/header.jsx:44]: @muskanbagrecha as far as performance is concerned, they both are same.

**Review by Shubhamkr2610 — APPROVED** [review]
Appreciation: your code is clean and readable , easy to understand . You are good to go

File: src/components/modal/LoginCardModal.jsx (hunk 3 above)
Comment (zoef-yt) [reply src/components/modal/LoginCardModal.jsx]: https://github.com/zoef-yt/react-ecom/pull/20/commits/98b8250aa90e0726b24de252cbcad7a704c54622

**Review by muskanbagrecha — APPROVED** [review]
Good to merge as you have implemented all the changes.

**Pull request merged**